printer_mods/README.md

```diff
@@ -51,7 +51,8 @@ like so:
 | |[V0 Neopixel Front Bed Mount](./Andre/V0_Neopixel_Front_Bed_Mount)|A replacement Front Bed Mount with room for a neopixel or dotstar behind the logo|:heavy_check_mark:|:x:|:x:|:x:|
 | |[Mini Afterburner ADXL345 Mount](./Andre/Mini_Afterburner_ADXL345_Mount)|Mount for an AXDL345 on the Mini Afterburner|:heavy_check_mark:|:x:|:x:|:x:|
 |antoinecellerier|[DS18B20 Sensor Mount](./antoinecellerier/ds18b20-sensor-mount)|DS18B20 temperature sensor mount|:heavy_check_mark:|:x:|:x:|:x:|:x:|
-|arkeet|[Dial test indicator mount](./arkeet/dti-mount)|A toolhead mount for a dial test indicator, where the tip is located at the nozzle. |:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|
+|arkeet|[Dial test indicator mount](./ArthurZheng/Foldable7InchDisplay)|A toolhead mount for a dial test indicator, where the tip is located at the nozzle. |:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|
+|ArthurZheng|[Foldable 7Inch Display](./arkeet/dti-mount)|Affordable foldable 7" touchscreen. |:x:|:x:|:heavy_check_mark:|:x:|
```

Comment:
The links are swapped here, `Foldable 7Inch Display` actually links to arkeet's Dial test indicator mount, and vice-versa.